lib/graphql.ts

```diff
@@ -25,6 +26,41 @@ export const GET_CONTESTS = gql`
   }
 `
 
+export const GET_CONTEST_DETAIL = gql`
```

Comment:
should be just `GET_CONTEST`, detail is just a way to call a component in the frontend